content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -175,6 +175,12 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+We have published the
+[JSON Schema](https://github.com/iterative/pipeline-schema) for `dvc.yaml` files
+that helps code editors and IDEs to provide auto-completion and
+validation/linting, and to verify the proper structure of `dvc.yaml` locally
+and/or in CI.
```

Comment:
Just submitting Ivan's suggestion from https://github.com/iterative/dvc.org/pull/1650#discussion_r464799496 formally:
```suggestion
💡 We maintain a `dvc.yaml`
[schema](https://github.com/iterative/pipeline-schema) that can be used
by editors like [VSCode](/doc/install/plugins#vscode) or
[PyCharm](/doc/install/plugins#pycharm) to enable automatic syntax
checks and auto-completion.
```
May need formatting. And the plugins.md sections don't exist yet.

Reply:
I'll commit this for now.